Loenn/lang/en_gb.lang

```diff
@@ -0,0 +1,43 @@
+# Mod Name
+mods.EverestYamlName.name=IsaGrabBag
```

Comment:
You probably forgot to change `EverestYamlName`, which is supposed to match the mod's `everest.yaml` name. Whatever you write after the equal sign is what is shown in Lönn.
```suggestion
mods.IsaGrabBag.name=Isa's Grab Bag
```